components/support/ffi/src/error.rs

```diff
@@ -252,6 +252,7 @@ impl From<Box<dyn std::any::Any + Send + 'static>> for ExternError {
         } else {
             "Unknown panic!".to_string()
         };
+        log::error!("Caught a panic calling rust code: {:?}", message);
```

Comment:
Is there any change of this including PII?

Reply:
Probably not. We already include this in the error message that gets sent with sentry, so it wouldn't be new PII if it is.